client/src/views/Dataset.vue

```diff
@@ -247,22 +212,21 @@ export default {
       this.unsavedDialogResolve(false);
       this.unsavedDialog = false;
     },
-    setRating(rating) {
-      if (rating !== this.rating) {
-        this.rating = rating;
-        this.ratingChanged();
+    setDecision(decision) {
+      if (decision !== this.decision) {
+        this.decision = decision;
+        this.onDecisionChanged();
       }
     },
     setNote(e) {
       this.newNote = e;
-      this.reviewChanged = true;
+      this.decisionChanged = true;
     },
-    async ratingChanged() {
-      if (!this.rating) {
-        this.reviewChanged = true;
+    async onDecisionChanged() {
+      if (this.decision != this.currentSession.decision) {
```

Comment:
Are we linting this code in CI? The linter should be forbidding `!=` hopefully. If not I'll add a ticket for linting in CI.

Reply:
That was passing `yarn lint` for some reason, so CI wouldn't have mattered.